Library/Formula/spread.rb

```diff
@@ -0,0 +1,65 @@
+require 'formula'
+
+class Spread < Formula
+  homepage 'http://www.spread.org'
+  url 'ftp://ftp4.gwdg.de/pub/FreeBSD/ports/local-distfiles/ohauer/spread-src-4.2.0.tar.gz'
+  mirror 'http://ftp.nsysu.edu.tw/FreeBSD/ports/local-distfiles/ohauer/spread-src-4.2.0.tar.gz'
+  sha1 '6052d39610ec27e856d601310f5b7a7b982d02dc'
+
+  option :universal
+
+  def install
+    ENV.universal_binary if build.universal?
+    system "./configure", "--prefix=#{prefix}", "--mandir=#{man}"
+    system "make"
+    system "make install"
+  end
+
+  def post_install
+      (var+"spread").mkpath
+  end
+
+  def caveats; <<-EOS.undent
+    A "/usr/local/etc/spread.conf" file exists. However, you must add an
+    entry to the file for your hostname and primary IP address. Failure
+    to do so will cause the program to abort with signal 6.
+    EOS
+  end
```

Comment:
Use `#{etc}` rather than `/usr/local/etc`. Also, is there no way we can configure this for the user?